pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -130,6 +138,12 @@ func (b *ConsoleServerCLIConfigBuilder) customization() Customization {
 	if len(b.docURL) > 0 {
 		conf.DocumentationBaseURL = b.docURL
 	}
+	if len(b.customProductName) > 0 {
+		conf.CustomProductName = string(b.customProductName)
```

Comment:
`customProductName` is already defined as string

Reply:
right, not sure why that ended up in here.